support/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md

```diff
@@ -7,23 +7,38 @@ description: "SparkPost supports HTTPS engagement tracking for customers via sel
 
+> Alternative: to configure HTTPS engagement tracking using your own proxy, see [this article](using-proxy-https-tracking-domain.md).
+
 ## Configuring SSL Certificates
 
 In order for HTTPS engagement tracking to be enabled on SparkPost, our service needs to present a valid certificate that will be trusted by the email recipient’s browser.  SparkPost does not manage certificates for customer engagement tracking domains, as we are not the record owner for our customers’ domains.
 
-Use a CDN like [Cloudflare](http://www.cloudflare.com) or [Fastly](http://www.fastly.com) to manage certificates and keys for any custom engagement tracking domains.  These services forward traffic onwards to SparkPost so that HTTPS tracking can be performed.
+Use a CDN like [Cloudflare](http://www.cloudflare.com), [Fastly](http://www.fastly.com) or [AWS Cloudfront](https://aws.amazon.com/cloudfront/) to manage certificates and keys for any custom engagement tracking domains.  These services forward traffic onwards to SparkPost so that HTTPS tracking can be performed.
 
 ## How to Create a Secure Tracking Domain on SparkPost
 
 In addition to SSL certificates, link forwarding, and page rules (see the step by step guide below), you will need to create a tracking domain with the tracking domains API using the `"secure": true` string. Detailed information on this operation can be found in our API documentation [here](https://developers.sparkpost.com/api/tracking-domains.html#tracking-domains-create-and-list-post).
 
-## How to Switch a Tracking Domain from Insecure to Secure
+## <a name = "switch-to-secure"></a> How to Switch a Tracking Domain from Insecure to Secure
 
 If you have previously created a tracking domain (whether verified or unverified), and wish to switch it from insecure (the default value for tracking domains) to secure, use the tracking domains API `PUT` call to update the tracking domain with the `"secure": true` string. Detailed information on this operation can be found in our API documentation [here](https://developers.sparkpost.com/api/tracking-domains.html#tracking-domains-retrieve,-update,-and-delete-put).
 
-## Step by Step Guide with CloudFlare
 
-The following is a sample guide for use with CloudFlare **only**; please note, the steps to configure your chosen CDN will likely differ from CloudFlare in workflow. Please refer to your CDN's documentation and contact their respective support departments if you have any questions.
+## Step by Step guides
+
+This document includes step by step guides for the following CDNs.
+
+* [CloudFlare](#cloudflare): create a Domain
```

Comment:
I would have these uniform layout wise.  It looks a bit off otherwise.

Reply:
There will be more sections in the AWS CloudFront part, but I'll adjust the layouts to match.